recipes/opentrackio-cpp/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.20)
```

Comment:
3.15 for all of conan center, remember there is no guarantee this will be higher than 3.15 for the test package